classes/classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :classroom
+
```

Comment:
- [ ] Kindly, make sure that you define the relationship between Classroom and Student. `attr_reader :students`

Reply:
@RolandM99 , 
Thank you for the invaluable review comments. But I don't understand what this means in a practical setting for me. I am doing my best to understand this, and a little more explanation can help.

Reply:
@RolandM99 I think I you mean I use this `attr_accessor :students` instead of `attr_reader :classroom`

Reply:
What I'm trying to tell you is just simple,  replacing `:classroom` with `students`. Actually in your code, it's kinda like you're defining getter for classroom ignoring the relationship with student.
So, just replace `:classroom` with `:students`